portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -67,16 +73,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Send the JSON as the response
     // response.setContentType("application/json;");
     // response.getWriter().println(json);
+
+    Query query = new Query("Message").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> messages = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
```

Comment:
Since you aren't using `id` or `timestamp`, you can probably delete those lines?